server/athenian/api/models/state/readd_goal_templates.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
You named the script "reaDD", the second D should be a typo.

Reply:
Right, 
I renamed to `reimport_goal_templates` seems more correct